scripts/airgapped/README.md

```diff
@@ -0,0 +1,85 @@
+# Airgap Utility Scripts
+
+This directory contains bash and python scripts that are useful for performing
+an airgapped installation. These scripts could either be used independently
+to create airgap artifacts or via our testing scripts.
+
+We'll document some use-case scenarios here for the different scripts.
+
+## Prerequisites
+
+To use the scripts in this directory you'll need to install a couple of Python
+and Ubuntu packages.
+```
+pip3 install -r requirements.txt
+sudo apt install pigz
+```
+
+## Get list of all images in bundle
+
+In a lot of cases we need to have the list of all OCI Images used by a bundle.
+To get this list we can use the following script. This is scripts makes the
```

Comment:
Not a mandatory change, but I think you just need to explain script usage. 
 You don't need to convince them it was important to make the script

```suggestion
Use `./scripts/airgapped/get-all-images.sh` to get the list of all OCI Images used by a bundle. This script makes the
```

Reply:
Added a commit for this, since the discussion was outdated due to other commits
https://github.com/canonical/bundle-kubeflow/pull/682/commits/afc70a9f5107ac25444c8bdba393def5640d41b2